src/components/TopPick/TopPick.tsx

```diff
@@ -0,0 +1,158 @@
+import React from "react";
+import { css } from "emotion";
+
+import { from } from "@guardian/src-foundations/mq";
+import { space, neutral, palette } from "@guardian/src-foundations";
+import { textSans } from "@guardian/src-foundations/typography";
+
+import { GuardianStaff } from "../Badges/Badges";
+import { CommentType } from "../../types";
+import { Avatar } from "../Avatar/Avatar";
+import { RecommendationCount } from "../RecommendationCount/RecommendationCount";
+import { Timestamp } from "../Timestamp/Timestamp";
+import { joinUrl } from "../../lib/joinUrl";
+
+type Props = { baseUrl: string; comments: CommentType[] };
+
+const pickStyles = css`
+  width: 100%;
+  min-width: 250px;
+  margin-bottom: ${space[5]}px;
+  flex: 0 0;
+  ${textSans.small()};
+`;
+
+const arrowSize = 25;
+const bg = neutral[93];
+
+const pickComment = css`
+  padding: ${space[3]}px;
+  background-color: ${bg};
+  border-radius: 15px;
+  margin-bottom: ${arrowSize + 5}px;
+  position: relative;
+
+  ${from.tablet} {
+    min-height: 150px;
+  }
+
+  :before {
+    content: "";
+    margin-left: 24px;
+    position: absolute;
+    border-right: ${arrowSize}px solid transparent;
+    border-top: ${arrowSize}px solid ${bg};
+    bottom: -${arrowSize - 1}px;
+  }
+`;
+
+const pickMetaWrapper = css`
+  display: flex;
+  justify-content: space-between;
+  padding-top: ${space[2]}px;
+`;
+
+const userDetails = css`
+  display: flex;
+  justify-content: space-between;
+`;
+
+const userMetaStyles = css`
+  display: flex;
+  flex-direction: column;
+`;
+
+const userName = css`
+  font-weight: bold;
+  color: ${palette.news.main}; /* TODO USE PILLAR */
```

Comment:
Could use news.main instead of palette

Reply:
I've got another ticket to make this colouring dynamic based on `pillar` so we need the full palette

Reply:
You could create your own little pillar colour dictionary. I'm not sure how much of a saving you'll get over importing the entire palette though.

```js
const pillarColours = {
  news: news[400],
  sport: sport[400],
  // ...
}
```